packages/gatsby-plugin-netlify/src/create-redirects.js

```diff
@@ -12,8 +12,16 @@ export default async function writeRedirectsFile(
 
   const FILE_PATH = publicFolder(`_redirects`)
 
-  // Map redirect data to the format Netlify expects
   // https://www.netlify.com/docs/redirects/
+  const NETLIFY_REDIRECT_KEYWORDS_ALLOWLIST = [
+    `query`,
+    `conditions`,
+    `headers`,
+    `signed`,
+    `edge_handler`,
+  ]
```

Comment:
https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-country-or-language
`Country` and `Language` entries should be included too

Reply:
Adding

Reply:
Done, thank you so much!